File: app/src/main/java/com/onedelay/mymovie/api/VolleyHelper.java
package com.onedelay.mymovie.api;

import com.android.volley.RequestQueue;

public class VolleyHelper {
    public static RequestQueue requestQueue;

    public static String host = "http://boostcourse-appapi.connect.or.kr";
    public static int port = 10000;

    public static void requestServer(GsonRequest request) {
        request.setShouldCache(false);
        if (requestQueue != null) {
            requestQueue.add(request);
        }
    }
}

File: app/src/main/java/com/onedelay/mymovie/data/api/VolleyHelper.kt
package com.onedelay.mymovie.data.api

import com.android.volley.RequestQueue

object VolleyHelper {
    var requestQueue: RequestQueue? = null

    var host = "http://boostcourse-appapi.connect.or.kr"
    var port = 10000

    fun requestServer(request: GsonRequest<*>) {
        request.setShouldCache(false)
        if (requestQueue != null) {
            requestQueue!!.add(request)
        }
    }
}
